apps/src/applab/designElements/elementUtils.js

```diff
@@ -3,9 +3,12 @@ module.exports.rgb2hex = function (rgb) {
   if (rgb === '') {
     return rgb;
   }
-  rgb = rgb.match(/^rgb\((\d+),\s*(\d+),\s*(\d+)\)$/);
```

Comment:
Maybe update the comment above the function to say something along the lines of this is based on that SO answer, but has since been modified.